LandSandBoat/server
# Nm mob fixes

**_I affirm:_**

## What does this pull request do?
**General NM fixes:**
Overhauled many NM's level and or HP value to be more in line with reported wikipedia values (boosted). Many lower level NMs didn't have the appropriate HP or level assigned.

**Shen Overhaul:**
1. Shen now enters and exits his shell in 45-120 second intervals
2. Shen and filtrates now have the appropriate spell lists
3. Fixed a bug where filtrates would spawn with Shen and continued to spawn after he died
4. Shen now has physical and magical resistances while in his shell, prevented his movement and attacks, and was given a regen
5. Fixed a bug where Shen would spawn inside his shell after previously being defeated in his shell
6. Adjusted filtrate's spawn points to be unique (slightly moved to the sides of Shen)
7. Filtrates now only spawn while Shen is in his shell
8. (Other) Should consider giving Shen a draw in effect to prevent players from exploiting the ability to pull shen away from filtrate's spawn points. Or dynamically spawning the filtrates next to Shen

**Voluptuous Vivian:**
Prevented Vivian's draw in from using draw in on the entire alliance, and reduced it to single target. Evidence of this can be seen in the following video: https://www.youtube.com/watch?v=mCtheeXMuHo&t=8s&ab_channel=PawkyFTW

**Other:**
**Wyvern / Dragoon fix:**
The wyvern will now correctly use super climb alongside the dragoon using super jump

**Beastmaster Pets:**
Removed drop tables from mob-beastmaster's pets

## Steps to test these changes
1. Battle Shen and test that his entrance and exit shell intervals are correct
2. Test that the filtrates only spawn while he is in his shell
3. Test resistances while in shell

1. Use super jump as a dragoon to trigger the wyvern to use super climb

1. Allow Voluptuous Vivian to use draw in on party to ensure only the target with enmity is drawn in.
